src/transformers/image_utils.py

```diff
@@ -118,7 +118,7 @@ def to_pil_image(self, image, rescale=None):
 
     def convert_rgb(self, image):
         """
-        Converts `image` to RGB format. Note that this will trigger a conversion of `image` to a PIL Image.
+        Converts `image` to RGB format.
```

Comment:
I would add here, that it only does convert to RGB in case the image is of type PIL.